seed/challenges/03-back-end-development-certification/git.json

```diff
@@ -8,67 +8,27 @@
       "id": "bd7353d8c341eddeaeb5bd0f",
       "title": "Save your Code Revisions Forever with Git",
       "description": [
-        "We'll build this Waypoint on Cloud 9, a powerful online code editor with a full Ubuntu Linux workspace, all running in the cloud.",
-        "If you don't already have Cloud 9 account, create one now at <a href='http://c9.io' target='_blank'>http://c9.io</a>.",
-        "Open up <a href='http://c9.io' target='_blank'>http://c9.io</a> and sign in to your account.",
-        "Click on the \"+\" icon at the top right of the c9.io page to create a new workspace.",
-        "Give your workspace a name and an optional description.",
-        "Choose Node.js in the selection area below the name field.",
-        "Click the \"Create workspace\" button.",
-        "Once C9 builds and loads your workspace, you should see a terminal window in the lower right hand corner. In this window use the following commands. You don't need to know what these mean at this point.",
-        "Install <code>git-it</code> with this command: <code>npm install -g git-it</code>",
-        "Before you start, read <a href='https://help.github.com/articles/adding-a-new-ssh-key-to-your-github-account/' target='_blank'>Adding a new SSH key to your GitHub account</a> if you have two-factor authentication set up on your GitHub account.",
-        "Now start the tutorial by running <code>git-it</code>.",
-        "Note that you can resize the c9.io's windows by dragging their borders.",
-        "Make sure that you are always in your project's \"workspace\" directory. You can always navigate back to this directory by running this command: <code>cd ~/workspace</code>.",
-        "You can view this Node School module's source code on GitHub at <a href='https://github.com/jlord/git-it' target='_blank'>https://github.com/jlord/git-it</a>.",
-        "Complete \"Get Git\"",
-        "Complete \"Repository\"",
-        "Complete \"Commit to it\"",
-        "Complete \"Githubbin\"",
-        "Complete \"Remote Control\"",
-        "Complete \"Forks and Clones\"",
-        "Complete \"Branches aren't just for Birds\"",
-        "Complete \"It's a Small World\"",
-        "Complete \"Pull, Never out of Date\"",
-        "Complete \"Requesting you Pull, Please\"",
-        "Complete \"Merge Tada!\"",
+        "To complete this challenge, you will need to install a few things on your computer - a text editor, Git, and the Git-It-Electron desktop app.",
+        "Text Editor - A text editor is a good tool to have as a programmer. You can download and install <a href=\"https://atom.io/\" target=\"_blank\">Atom</a>, <a href=\"https://www.sublimetext.com/3\" target=\"_blank\">Sublime Text</a>, or <a href=\"http://brackets.io/\" target=\"_blank\">Brackets</a>. All these editors are available for Windows, Mac and Linux platforms. Alternatively, if you already have a text editor, or you would like to use a different one, you're welcome to do so.",
```

Comment:
Yup, I guess that would not hurt. But again, I'll leave @dhcodes to add them at his will. The choice is left to users in any case. And the ones above have the two most used and popular ones as well. Again it's just a matter of personal choice.